cluster/src/main/java/org/apache/iotdb/cluster/config/ClusterConfig.java

```diff
@@ -184,6 +184,24 @@ public ClusterConfig() {
     seedNodeUrls = Arrays.asList(String.format("%s:%d", internalIp, internalMetaPort));
   }
 
+  /**
+   * If the path is not updated frequently, you can enable cache to update the path every 1s
+   * improving the query performance by 14%
+   */
+  private boolean enableQueryPathsCache = false;
+
+  /**
+   * When enable_query_paths_cache is true, this parameter effective. query paths cache size,
```

Comment:
grammer error? `this parameter effective`
what's this ? `query paths cache size`

Reply:
done